targets/TARGET_STM/TARGET_STM32F3/device/stm32f3xx_hal_pcd.h

```diff
@@ -622,9 +622,9 @@ PCD_StateTypeDef HAL_PCD_GetState(PCD_HandleTypeDef *hpcd);
   * @retval None
   */
 #define PCD_CLEAR_RX_EP_CTR(USBx, bEpNum)   (PCD_SET_ENDPOINT((USBx), (bEpNum),\
-                                   PCD_GET_ENDPOINT((USBx), (bEpNum)) & 0x7FFFU & USB_EPREG_MASK))
+                                   ( PCD_GET_ENDPOINT((USBx), (bEpNum)) | USB_EP_CTR_TX ) & ~USB_EP_CTR_RX & USB_EPREG_MASK))
```

Comment:
LGTM but I need to check with the original writter of this file to see if it is 100% safe.
Can you add a comment at the end of the line like: "// MBED fix". This will help when we will update the Cube HAL files with newer version.